index.html

```diff
@@ -0,0 +1,172 @@
+          </li>
+        </ul>
+      </div>
+
+      <div class="anchor-arrow">
+        <img src="images/anchor-arrow.png" alt="arrow" />
+      </div>
+    </section>
+    <section class="project-container">
+      <div class="project-title">
+        <h2>My Recent </h2>
+      </div>
+      <div class="project-list">
+        <ul>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Tonic</h2>
+              <div class="buttons">
+                <div>PHP</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="buttons">
+                <div>Ruby on rails</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="buttons">
+                <div>Ruby on rails</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <div class="about-me">
+      <section class="about-container">
+        <header class="about-header">
+          <h2>About me</h2>
+        </header>
+        <div class="about-style">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have
+            a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="get-resume-button">
+          <button type="button">Get My Resume</button>
+        </div>
+        <div class="skills-container">
+          <ul>
+            <li class="item-list">
+             
+              <div>
+                <h2>Languages</h2>
+              </div>
+              <div class="about-buttons">
+                <div>Ruby on Rails</div>
+                <div>Ruby</div>
+                <div>Html</div>
+                <div>CSS</div>
+              </div>
```

Comment:
- [ ] For this list it would be great if you used `ul` and `li` tags instead of `div` tags.